cmd/broadcaster-cli/app/utxos/broadcast/broadcast.go

```diff
@@ -109,43 +103,34 @@ var Cmd = &cobra.Command{
 			return fmt.Errorf("failed to create client: %v", err)
 		}
 
-		fundingKeySets := make([]*keyset.KeySet, len(keyFiles))
-		for i, kf := range keyFiles {
+		wg := &sync.WaitGroup{}
```

Comment:
What is the use of this wait group where we just once `Add(1)` and never call `Done()`?
`wg.Wait()` will wait forever